docs/release-notes/release-notes-0.17.0.md

```diff
@@ -47,6 +47,9 @@ unlock or create.
 * [Restore support](https://github.com/lightningnetwork/lnd/pull/7678) for
   `PKCS8`-encoded cert private keys.
 
+* [Remove getblockhash](https://github.com/lightningnetwork/lnd/pull/7699) 
+  from the neutrino rpc sub-server.
```

Comment:
maybe add a comment to mention they can use the other `getblockhash`.

Reply:
Good call.